### PR TITLE
bpo-38602: Add fcntl.F_OFD_XXXX for fcntlmodule

### PR DESCRIPTION
**Pull request by corona10**
https://bugs.python.org/issue38602

### CODE REVIEW

File: Doc/whatsnew/3.9.rst [hunk 1]
```diff
@@ -125,8 +125,15 @@ that schedules a shutdown for the default executor that waits on the
 :func:`asyncio.run` has been updated to use the new :term:`coroutine`.
 (Contributed by Kyle Stanley in :issue:`34037`.)
 
+fcntl
+-----
+
+:data:`~fcntl.F_OFD_GETLK`, :data:`~fcntl.F_OFD_SETLK` and
+:data:`~fcntl.F_OFD_SETLKW` are added for fcntlmodule.
```
Comment (serhiy-storchaka) [inline Doc/whatsnew/3.9.rst]: There is no "fcntlmodule". And mentioning the "fcntl" module would be redundant. I suggest:
```
Added constants :data:`~fcntl.F_OFD_GETLK`, :data:`~fcntl.F_OFD_SETLK` and :data:`~fcntl.F_OFD_SETLKW`.
```

File: Misc/NEWS.d/next/Library/2019-10-27-22-29-45.bpo-38602.7jvYFA.rst [hunk 2]
```diff
@@ -0,0 +1,2 @@
+:data:`~fcntl.F_OFD_GETLK`, :data:`~fcntl.F_OFD_SETLK` and
```
Comment (serhiy-storchaka) [inline Misc/NEWS.d/next/Library/2019-10-27-22-29-45.bpo-38602.7jvYFA.rst]:
```
Added constants ... to the :mod:`fcntl` module.
```

**Review by serhiy-storchaka — COMMENTED** [review]
Add also the versionchanged directive in the module documentation.

**Comment by corona10** [comment]
@serhiy-storchaka 

Thanks, I've updated the PR.

**Review by serhiy-storchaka — APPROVED** [review]

**Pull request merged**